frontend/src/plugins/impl/ButtonPlugin.tsx

```diff
@@ -38,6 +40,7 @@ export class ButtonPlugin implements IPlugin<number, Data> {
         variant={kindToButtonVariant(kind)}
         disabled={disabled}
         size="xs"
+        keyboardShortcut={keyboardShortcut}
```

Comment:
we could also update `<Tooltip` below to be `<Tooltip delay={200}/>` it is more apparent / easier to signify the shortcut

Reply:
you can also use `<KeyboardHotkeys shortcut={keyboardShortcut}/>` to render the shortcut in the tooltip